tensorflow/core/api_def/base_api/api_def_RegexMatch.pbtxt

```diff
@@ -0,0 +1,22 @@
+op {
+  graph_op_name: "RegexMatch"
+  in_arg {
+    name: "input"
+    description: <<END
+The text to be processed.
+END
+  }
+  in_arg {
+    name: "pattern"
+    description: <<END
+The regular expression to match the input.
+END
+  }
+  out_arg {
+    name: "output"
+    description: <<END
+The same tensor shape as `input`.
+END
+  }
+  summary: "Check if the input matches the regex pattern."
```

Comment:
Can you expand the documentation to describe the shapes and nature of the output. As I read it, the op takes a string tensor of any shape, and a scalar string tensor containing a single regex pattern. It returns a boolean tensor that of the same shape as the input, with values True for those inputs that match the pattern. Perhaps say that this expect regular expressions in the default format used by RE2 and give a link to the description of this.

Reply:
Thanks @rmlarsen. The doc has been updated.